Jint/Native/Array/ArrayInstance.cs

```diff
@@ -289,8 +288,12 @@ public override void RemoveOwnProperty(string p)
 
         public static bool IsArrayIndex(JsValue p, out uint index)
         {
-            index = ParseArrayIndex(TypeConverter.ToString(p));
+            return IsArrayIndex(TypeConverter.ToString(p), out index);
+        }
 
+        private static bool IsArrayIndex(string p, out uint index)
```

Comment:
better signature when called with string, does not cause automatic JsValue conversion